pkg/ddl/ddl.go

```diff
@@ -1563,9 +1563,13 @@ type RecoverInfo struct {
 type RecoverSchemaInfo struct {
 	*model.DBInfo
 	RecoverTabsInfo []*RecoverInfo
-	DropJobID       int64
-	SnapshotTS      uint64
-	OldSchemaName   model.CIStr
+	// RecoverSchemaIDOnOwner is the new logic to avoid a large RecoverTabsInfo can't
+	// be persisted. If it's set, DDL owner will recover RecoverTabsInfo instead of
+	// job submit node.
+	RecoverSchemaIDOnOwner int64
```

Comment:
seems this id duplicate with job.SchemaID and `*model.DBInfo.ID` in this struct, maybe add a `LoadOnExecute bool`?